### ISSUE TITLE
Counting correct and incorrect answers in MC Questions?

### ISSUE THREAD

**Issue by petzi53** [rstudio/learnr#166]
When developing a battery of MCQs it would be nice to have a possibility to count correct and incorrect answers. I have tried to add a counting variable in the correct and incorrect string, e.g.

`correct = paste("Correct: You've reached", my_good_answers <- my_good_answers + 1, " of 22 possible points.")`

But this approach did not work out as the programme seems to run the code always during compilation. Is there another possibility?

**Comment by jasonbaik94** [comment]
Hi Peter, It seems like we have similar problems. I've posted mine on StackOverflow [https://stackoverflow.com/questions/53093850/how-do-i-insert-conditions-in-learnr](url) 

Have you found a solution?

**Comment by petzi53** [comment]
No, sorry. Because we didn't get an answer, I think that our problem is not solvable at the moment. Hopefully the developer keeps our questions in mind for a future version. To know how good the student answered the test is a very important educational issue.

**Comment by jasonbaik94** [comment]
Yes, keeping my fingers crossed.
